### ISSUE TITLE
Pyrevit run commandline option:  to run the script in already opened instance

### ISSUE THREAD

**Issue by chanda-korat** [pyrevitlabs/pyRevit#2290]
I am using pyrevit run commandline option to perform some activities on revit model, but pyrevit opens new instance and closes the file and app after execution. 
My requirement is to do other operations in between actions, so I want to keep the app open and also sometimes want to capture the open instance. Is there any option to achieve this? 
I tried to run the script externally directly using IronPython after manually opening Revit. But in that I am not able to capture running instance of application. HOST_APP, __revit__ variables only works within Revit environment so the same is not working.
Any suggestions will be really helpful and appreciated.

**Comment by sanzoghenzo** [comment]
Hi @chanda-korat, the pyrevit run command is designed to be able to run script in an automated way without user interacition (from what I get from the docs, I honestly never used it :sweat_smile:)

I don't quite get what are you trying to do:

- what are the "other operations in between actions"?
- what are the "actions" you want to perform?

Revit API cannot be controlled from outside of the revit process (as one could do with AutoCAD with the COM API - Revit doesn't have them).
I suppose your best bet would be to use pyRevit's [HTTP API Routes](https://pyrevitlabs.notion.site/pyRevit-Routes-HTTP-API-79ef6d4a77b04aca9be6f4e46ffa728e)
Basically you create API endpoints using pyRevit facilities and then in your "controller" program outside revit call those endpoints via an HTTP client such as requests, httpx (if you stay in the Python realm, but there are plenty other clients for all the languages)
